exporter_test.go

```diff
@@ -320,6 +321,88 @@ func testExporter(t *testing.T, when spec.G, it spec.S) {
 				h.AssertEq(t, meta.Stack.RunImage.Mirrors, []string{"registry.example.com/some/run", "other.example.com/some/run"})
 			})
 
+			when("metadata.toml does not include BOM", func() {
+				it.Before(func() {
+					err := ioutil.WriteFile(filepath.Join(layersDir, "config", "metadata.toml"), []byte(`
+[[processes]]
+  type = "web"
+  command = "npm start"
+`),
+						os.ModePerm,
+					)
+					h.AssertNil(t, err)
+				})
+
+				it("BOM is not present in that label", func() {
+					h.AssertNil(t, exporter.Export(layersDir, appDir, fakeAppImage, fakeImageMetadata, additionalNames, launcherPath, stack))
+
+					metadataJSON, err := fakeAppImage.Label("io.buildpacks.build.metadata")
+					h.AssertNil(t, err)
+
+					assert.JSONEq(t, `
+{
+  "bom": null
+}
+`,
+						metadataJSON)
+
+				})
+			})
+
+			when("metadata.toml includes BOM", func() {
+				it.Before(func() {
+					err := ioutil.WriteFile(filepath.Join(layersDir, "config", "metadata.toml"), []byte(`
+[[processes]]
```

Comment:
Not a big fan of breaking the indention here. I know we do it elsewhere, so not blocking the merge for this.

Reply:
What do you mean by breaking indentation? 

Are you referring to adding a new line character after the backtick? or are you saying you would prefer we don't inline a multiline string as method arguments?